packages/web-components/src/components/footer/legal-nav.ts

```diff
@@ -17,6 +17,26 @@ import styles from './footer.scss';
 const { prefix } = settings;
 const { stablePrefix: ddsPrefix } = ddsSettings;
 
+/**
+ * Footer size.
+ */
+export enum FOOTER_SIZE {
+  /**
+   * Regular size.
+   */
+  REGULAR = '',
+
+  /**
+   * Short size.
+   */
+  SHORT = 'short',
+
+  /**
+   * Micro size.
+   */
+  MICRO = 'micro',
+}
```

Comment:
Sorry I was not clear enough; Would you update `footer.ts` to add the micro variant and import the num from `footer.ts` to here? Same for the other file.